### PR TITLE
User model

### PR DESCRIPTION
**Pull request by nmmascia**

### CODE REVIEW

File: app/api/users.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module Tracker
+  class Users < Grape::API
+    format :json
+
+    resource :users do
+      desc "Returns all users"
+      get "/" do
+        User.all
+      end
```
Comment (BrianBorge) [inline app/api/users.rb]: I don't we should have an endpoint for all users.

Comment (nmmascia) [inline app/api/users.rb]: all the usersssss!

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,9 @@
 
-require ::File.expand_path('../config/environment', __FILE__)
+require ::File.expand_path("../config/environment", __FILE__)
+
+models = Dir::open("./app/models").map do |filename|
+  filename.gsub!(".rb", "").to_sym if filename.include? ".rb"
+end
```
Comment (BrianBorge) [inline Rakefile]: I think you will want to use ActiveSupport::Inflector.pluralize here.  Reason being is that model file in /models is singular but the sql tables are plural.


See example from https://github.com/BrianBorge/Tracker/blob/master/config/database.rb below:

```ruby
Dir[APP_ROOT.join('app', 'models', '*.rb')].each do |model_file|
  filename = File.basename(model_file).gsub('.rb', '')
  autoload ActiveSupport::Inflector.camelize(filename), model_file
end
```

File: Rakefile [hunk 3]
```diff
@@ -14,31 +18,42 @@ namespace :db do
...
+    end
   end
 
-  desc "Drop Workouts Table at #{DB_NAME}"
+  desc "Drop all tables at #{DB_NAME}"
   task :drop_table do
-    puts 'Dropping Workouts Table'
-    DB.drop_table(:workouts)
+    puts "Dropping all tables"
+    models.each { |table| DB.drop_table(table) }
```
Comment (BrianBorge) [inline Rakefile]: See above comment about `ActiveSupport::Inflector `.

File: Rakefile [hunk 4]
```diff
@@ -14,31 +18,42 @@ namespace :db do
...
-    Workout.insert(name: 'moop')
-    Workout.insert(name: 'mapp')
+    puts "Inserting into Workouts"
+    Workout.insert(name: "meep")
+    Workout.insert(name: "moop")
+    Workout.insert(name: "mapp")
+
+    puts "Inserting into Users"
+    User.insert(username: "nmmascia", password_digest: "password")
+    User.insert(username: "ballinborge", password_digest: "password")
```
Comment (BrianBorge) [inline Rakefile]: We'll move the creation of tables to use sequel migrations and inserts to the seed file soon.

**Pull request merged**